packages/algob/test/internal/deployer.ts

```diff
@@ -145,8 +153,9 @@ describe("DeployerDeployMode", () => {
       creator: "addr-1-get-address-dry-run",
       txId: "tx-id-dry-run",
       confirmedRound: -1,
-      appID: -1,
-      timestamp: 1
+      appID: 33,
```

Comment:
could you please explain why we're changing appId here?

Reply:
because teal doesn't support negative numbers, it is needed when making tx